src/game/game.go

```diff
@@ -0,0 +1,43 @@
+package game
+
+type Grid struct {
+	Data [][]int
+	Size int
+}
+
+func AreGridEquals(grid Grid, grid2 Grid) bool {
```

Comment:
s/AreGridEquals/AreGridsEquals

Reply:
For information, you can add methods to structs in Golang too but it's far from functional programming style.. It's still good to know :) 

It can let you do this kind of thing for example:

`myGrid.equals(myOtherGrid))`

https://gobyexample.com/methods